java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -398,12 +398,13 @@ public final ColumnVector isInteger() {
    * for null entries.
    *
    * @param intType the data type that should be used for bounds checking. Note that only
-   *                integer types are allowed.
+   *                integer types are allowed including (int8, int16, int32, int64).
```

Comment:
This is not correct and that type list may not be needed. You missed unsigned types.

Reply:
From a Java developers point of view integer types will not include bytes, shorts or longs. I am trying to highlight to the dev that this isn't just Java int its a cpp/cudf int. which does include signed/unsigned int8 - int64. 
How does this sound `Note that only cudf integer types are allowed including signed/unsigned int8 through int64`

Reply:
Yes that is great.